system-design/margo-overview/personas.md

```diff
@@ -1,37 +1,70 @@
+- Markets and sells their products to end users
 
-Note:
-Application Marketplaces are out of scope for Project Margo. However, Margo provides guidelines around content required within the Application Package that is utilized by Marketplaces.  
+## Persona Definitions
 
-**Workload Orchestration Vendor** 
+These persona definitions represent individual entities with specified use cases and requirements, aligned with the other personas in their category.
 
-The Workload Orchestration Vendor provides End Users with the infrastructure to manage applications along with Kubernetes Node and Node Cluster enrollment and management.   
+### End Users
 
-**Device Owner**
-The Device Owner within Project Margo is responsible for meeting the requirements outlined within the device requirements section.
+End users consume products and technologies from the suppliers to drive business continuity and efficiency
 
-[Device Requirements](../device-interoperability/device-requirements.md)
+#### OT User
 
-**Device Manufacturer**
+Consumer of functionality provided by the application vendors to run critical and non-critical business functions, or to improve business efficiency
 
-The Device Manufacturer within Project Margo is responsible for providing hardware that complies with the Project Margo device requirements that enable End Users within the Margo ecosystem. The Device Manufacturer is responsible for managing the device's lifecycle for the End-User. An additional option is for the Manufacturer to contract a Device Integrator to manage the lifecycle for the End-User. 
+#### IT Service Provider
 
-**Device Integrator**
+Provides "IT-like" services, such as connectivity, backup and restore, automation, security and auditing, etc, at edge sites
 
-The Device Integrator within Project Margo is responsible for transforming a generic device into a Project Margo device. Once the device is Margo compliant, the Device Integrator is responsible for managing the device's lifecycle for the End-User.
+#### End User Architect
 
-**Device Orchestration Vendor**
+Creates and enforces standards across deployment locations or sites for greater supportability and consistency
 
-The Device Orchestration Vendor provides End Users with the service to manage their Edge Devices lifecycle within the Margo ecosystem.  
+#### Integrator
+
+Optional persona, external to the organization of the other end user personas, but tasked with assembling and installing hardware and software provided by the suppliers
+
+### Suppliers
+
+Suppliers provides hardware or software that's evaluated and deployed by the end users
+
+#### Workload Supplier
+
+Provides an application that performs some desired function, such as computer vision, software-defined control, etc, which is deployed to devices by an orchestrator
+
+#### Orchestrator Supplier
+
+Provides a software package that ingests application definitions into a catalog, and allows an end user to deploy that application to devices
```

Comment:
This does not sound to me like an orchestrator. It sounds to me more like a "publisher".